src/module/actor/actions/single-check.ts

```diff
@@ -155,6 +156,18 @@ class SingleCheckActionVariant extends BaseActionVariant {
                     note.selector ||= selector; // treat empty selectors as always applicable to this check
                     return note;
                 }),
+            target: options.target
+                ? () => {
```

Comment:
The outer check for the presence of a `target` field in options is unnecessary. Just always have the function and let it return `null` if the field is missing.

Reply:
Removed external check